images/quay-upload/image-upload.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
```

Comment:
This is just a script, not an image. It should be moved someplace else, and in a folder that makes it clear that it not used for operating the service.

Reply:
Actually, it is going to be an image when we move to PaC. For now, I had just written the script. But I will write the Dockerfile as well in this PR.
(just like all other images/scripts under this folder, we would require some modifications with the script to make it work with OpenShift Pipelines)

Reply:
I would still move this image to `test` (while waiting for the refactoring of the repo we talked we should do) in the meantime, as this image is not used by operators.

Reply:
Done. Moved it to test folder.